src/main/java/com/hazelcast/commandline/HazelcastCommandLine.java

```diff
@@ -15,75 +15,146 @@
+        return new PrintWriter(new OutputStreamWriter(printStream, StandardCharsets.UTF_8));
     }
 
     @Override
     public void run() {
     }
+
+    @Command(description = "Starts a new Hazelcast IMDG member", mixinStandardHelpOptions = true, sortOptions = false)
+    public void start(
+            @Option(names = {"-c", "--config"}, paramLabel = "<file>", description = "Use <file> for Hazelcast configuration. "
+                    + "Accepted formats are XML and YAML. ")
+                    String configFilePath,
+            @Option(names = {"-p", "--port"}, paramLabel = "<port>",
+                    description = "Bind to the specified <port>. Please note that if the specified port is in use, "
+                            + "it will auto-increment to the first free port. (default: 5701)", defaultValue = "5701")
+                    String port,
+            @Option(names = {"-i", "--interface"}, paramLabel = "<interface>",
+                    description = "Bind to the specified <interface> (default: bind to all interfaces).", defaultValue = "127.0.0.1")
+                    String hzInterface,
+            @Option(names = {"-j", "--jar"}, paramLabel = "<path>", split = ",", description = "Add <path> to Hazelcast "
+                    + "classpath (Use ',' to separate multiple paths). You can add jars, classes, or the directories that contain classes/jars.")
+                    String[] additionalClassPath,
+            @Option(names = {"-J", "--JAVA_OPTS"}, paramLabel = "<option>", parameterConsumer = JavaOptionsConsumer.class,
+                    description = "Specify additional Java <option> (Use ',' to separate multiple options).")
+                    List<String> javaOptions,
+            @Option(names = {"-v", "--verbose"},
+                    description = "Output with FINE level verbose logging.")
+                    boolean verbose,
+            @Option(names = {"-vv", "--vverbose"},
+                    description = "Output with FINEST level verbose logging.")
+                    boolean finestVerbose)
+            throws IOException, InterruptedException {
+        List<String> args = new ArrayList<>();
+        if (!isNullOrEmpty(configFilePath)) {
+            args.add("-Dhazelcast.config=" + configFilePath);
+        } else {
+            args.add("-Dhazelcast.default.config=" + AbstractCommandLine.WORKING_DIRECTORY + "/config/hazelcast.yaml");
+        }
+        args.add("-Dnetwork.port=" + port);
+        args.add("-Dnetwork.interface=" + hzInterface);
+        if (javaOptions != null && javaOptions.size() > 0) {
+            args.addAll(javaOptions);
+        }
+        args.add("-Djava.net.preferIPv4Stack=true");
+        addLogging(args, verbose, finestVerbose);
+
+        buildAndStartJavaProcess(HazelcastMember.class, args, additionalClassPath);
+    }
+
+    private void buildAndStartJavaProcess(Class aClass, List<String> parameters, String[] additionalClassPath)
```

Comment:
Consider removing `aClass` parameter. You always use `HazelcastMember.class`.

Reply:
👍